addons/account/controllers/portal.py

```diff
@@ -27,13 +26,16 @@ def _invoice_get_page_view_values(self, invoice, access_token, **kwargs):
             'invoice': invoice,
         }
         return self._get_page_view_values(invoice, access_token, values, 'my_invoices_history', False, **kwargs)
+    
+    def _get_invoices_domain(self):
```

Comment:
Shouldn't be top of `_prepare_home_portal_values()`?

Reply:
At best it is personal preferences no? I don't see a reason why I'd move it to the top?

Reply:
Ok. IMO this is generic method so it should be declare at the top of used. In this PR, we called it at line 14 and declared it at line 30.

Reply:
Honestly, you can find hundreds of samples like this in Odoo.